File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxx = 0
        count = 0
        for i in nums:
            if i == 1:
                count += 1
            else:
                maxx = max(maxx , count)
                count = 0
        
        maxx = max(maxx , count)
        return maxx

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int count = 0;
        int maxx = 0;

        for(int i : nums){
            if(i == 1){
                count++;
            }
            else{
                if(count>maxx){
                    maxx = count;
                }
                count = 0;
            }
        }

        if(count > maxx){
            maxx = count;
        }
        return maxx;
    }
};